prisma/helpers.ts

```diff
@@ -15,3 +16,13 @@ export async function cleanDatabase(providedPrisma?: PrismaClient) {
     await prisma.$executeRawUnsafe(`TRUNCATE TABLE "${tablename}" RESTART IDENTITY CASCADE;`)
   }
 }
+
+export const encrypt = (text: string, secretKey: string) => {
+  const key = crypto.scryptSync(secretKey, 'salt', 32)
```

Comment:
Why do we run `scryptSync` on the secret key instead of just passing it to `createCipheriv`?

Reply:
I think it's because `crypto.scryptSync` ensures the `secretKey` is the correct length and securely derived for AES-256 encryption.